src/main/resources/jupyter/startup.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+set -e -x
+
+##
+# This is a startup script designed to run on Leo-created Dataproc clusters.
+#
+# It starts up Jupyter and Welder processes. It also optionally deploys welder on a
+# cluster if not already installed.
+##
+
+# Templated values
+export GOOGLE_PROJECT=$(googleProject)
+export CLUSTER_NAME=$(clusterName)
+export OWNER_EMAIL=$(loginHint)
+export JUPYTER_SERVER_NAME=$(jupyterServerName)
+export WELDER_SERVER_NAME=$(welderServerName)
+export NOTEBOOKS_DIR=$(notebooksDir)
+export WELDER_ENABLED=$(welderEnabled)
+export DEPLOY_WELDER=$(deployWelder)
+export WELDER_DOCKER_IMAGE=$(welderDockerImage)
+
+if [ "$DEPLOY_WELDER" == "true" ] ; then
+    echo "Deploying Welder on cluster $GOOGLE_PROJECT / $CLUSTER_NAME..."
+    export WELDER_ENABLED="true"
+
+    # Run welder-docker-compose
+    gcloud auth configure-docker
+    docker-compose -f /etc/welder-docker-compose.yaml up -d
+
+    # Set EVs inside Jupyter container necessary for welder
+    docker exec -it $JUPYTER_SERVER_NAME bash -c "echo $'export WELDER_ENABLED=true\nexport NOTEBOOKS_DIR=$NOTEBOOKS_DIR' >> /home/jupyter-user/.bashrc"
+
+    # Move existing notebooks to new notebooks dir
+    docker exec -it $JUPYTER_SERVER_NAME bash -c "ls -I jupyter.log -I localization.log -I notebooks /home/jupyter-user | xargs -d '\n'  -I file mv file $NOTEBOOKS_DIR"
```

Comment:
This `mv`s everything under `~jupyter-user` to `~jupyter-user/notebooks`. I didn't want to `cp` because we could potentially run out of disk (depending what data files the user has on their cluster). It's a little scary `mv`ing users' files -- we should definitely test to make sure there are no edge cases. This could also break code that has absolute paths, but I'm not sure we can avoid that.